wowchemy/archetypes/slides/index.md

```diff
@@ -12,7 +12,42 @@ slides:
   theme: black
   # Choose a code highlighting style (if highlighting enabled in `params.toml`)
   #   Light style: github. Dark style: dracula (default).
-  highlight_style: dracula
+  #   Available highlight themes listed in: https://highlightjs.org/static/demo/
+  #   Use lower case names and replace space with hyphen '-'
+  highlight_style: monokai
+
+  mermaid_enabled: true # configure js or not
+  mermaid_options: # see mermaid documentation for other options
+    theme: base # default/base/dark/neutral/forest
+    securityLevel: loose
```

Comment:
Let's use the same security level as already used in Wowchemy JS and not expose this to the end user.

Reply:
Wowchemy uses defaults .. it was changed in 8.2 to prevent clickable node unless you set security level to loose. Which was the default prior to 8.2.

Let's make it configurable but keep the default as strict

Reply:
I don't follow why you suggest to expose the Mermaid security level to the user editing the Markdown slides - please can you clarify?

What is the issue with just setting a fixed Mermaid security level in the code?